### ISSUE TITLE
Correct LRS response to POST when Content-Type is application/x-www-form-urlencoded but no method parameter is included

### ISSUE THREAD

**Issue by garemoko** [adlnet/xAPI-Spec#919]
If the client makes a POST Statements request with a Content-Type of application/x-www-form-urlencoded but no method parameter, how should the LRS respond?

Currently Watershed will assume a method of GET and return statements. Learning Locker will assume a content type of application/json and save the statement. 

IMHO I think we should return a 400 error in this case. I don't see anywhere in the spec that requires a content type of application/json though, except for the statements part of multipart requests.

**Comment by DavidTPate** [comment]
Yeah, I see the implication that `application/json` should be supported, but nothing that other content types shouldn't be supported.

[Here](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI.md#lrs-requirements-2) for example mentions:
> When receiving a PUT or POST with a document type of "application/json”, an LRS MUST accept batches of Statements which contain no attachment Objects.

The [LRS Conformance](https://github.com/adlnet/xAPI_LRS_Test/blob/1d684b7798f05a2f11f916c441c43dd4ba65486c/TestingRequirements.md) doesn't mention anything about it either.

**Comment by brianjmiller** [comment]
Could also interpret it as `method=POST`.

**Comment by bscSCORM** [comment]
The way Learning Locker is handling it seems to be the most reasonable approach, since we don't call out a requirement for the content type to be `application/json`.

**Comment by garemoko** [comment]
Wax returns 400

**Comment by garemoko** [comment]
From the call 4th May, error 400 is the best approach. We need to use SHOULD* rather than MUST because the spec doesn't explicitly require a content type of application/json.

Something along the lines of:

The LRS SHOULD* reject any request with error code 400 where the content type header does not match the content included in the request or where the structure of the request does not match the structure outlined in this specification for a particular content type. For example, if the content of the request is formatted as JSON, the content type is expected to be application/json. If the content type is pplication/x-www-form-urlencoded it is expected that the request will include a method parameter as outlined in Alternate Request Syntax (Insert link).

**Comment by DavidTPate** [comment]
Sounds good @garemoko we likely should have something in there around the `Accept` header as well. I didn't see any mention of requests that send `Accept: application/xml` or really anything except `application/json` as the `Accept` header.

**Comment by garemoko** [comment]
@DavidTPate do you want to raise that as a separate issue? I think that needs more discussion.
